recipe/meta.yaml

```diff
@@ -36,10 +46,9 @@ outputs:
   - name: nvmath-python
     version: {{ version }}
     build:
-      number: {{ number }}
-      string: cuda{{ cuda_major }}_py{{ py }}h{{ PKG_HASH }}_{{ number }}  # [cuda_compiler_version != "None"]
-      string: core_py{{ py }}h{{ PKG_HASH }}_{{ number }}  # [cuda_compiler_version == "None"]
-      skip: true  # [(py < 310) or cuda_compiler_version not in ("None", "11.8", "12.0")]
+      number: {{ build_number }}
+      string: {{ build_string }}_py{{ py }}h{{ PKG_HASH }}_{{ build_number }}
+      skip: true  # [(py < 310) or cuda_compiler_version not in ("None", "11.8", "12.6")]
```

Comment:
btw I don't think it's possible but it'd be nice if conda-forge has a way for us to express easily "give me a CUDA 11 or 12 compiler, I don't care about the minor version" here, because nvmath-python really doesn't care...

Reply:
It is possible. We could change the channel pinnings from "12.6" to "12". I think it's really a legacy holdover from pre-cuda-enhanced-compatability days when there was not good compatibility between minor versions of the CTK. If the pinnings were changed, then every recipe would just automatically float up to the newest CTK.

Reply:
That'd be cool!